cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -313,7 +313,7 @@ func (s *CMServer) Run(_ []string) error {
 	// If apiserver is not running we should wait for some time and fail only then. This is particularly
 	// important when we start apiserver and controller manager at the same time.
 	var versionStrings []string
-	err = wait.PollImmediate(time.Second, 10*time.Second, func() (bool, error) {
+	err = wait.Poll(time.Second, 10*time.Second, func() (bool, error) {
 		versionStrings, err = client.ServerAPIVersions(kubeconfig)
 		return err == nil, err
```

Comment:
@ddysher hopefully what you wrote has a typo, if not please take a look at https://github.com/kubernetes/kubernetes/blob/95b265390e530eb137beac4288ba28d63c1142da/pkg/util/wait/wait.go#L95